migrations/V103__Update_Enums_Biannually_To_Semiannually.sql

```diff
@@ -0,0 +1,16 @@
+-- Update the columns that use the old enum values to use the new ones
+UPDATE plan_beneficiaries
+SET beneficiary_selection_frequency = 'SEMIANNUALLY'
+WHERE beneficiary_selection_frequency = 'BIANNUALLY';
+UPDATE plan_participants_and_providers
+SET provider_addition_frequency = 'SEMIANNUALLY'
+WHERE provider_addition_frequency = 'BIANNUALLY';
+UPDATE plan_ops_eval_and_learning
+SET data_sharing_frequency = ARRAY ['SEMIANNUALLY']::data_frequency_type[]
+WHERE 'BIANNUALLY' = ANY (data_sharing_frequency);

```

Comment:
Will this only update the entries from `BIANUALLY` to `SEMIANUALLY`, or will it set the entire array to one with a single entry of `SEMIANUALLY`?

It looks like it'd do the latter, but I haven't tested that - just looking at `SET data_sharing_frequency = ARRAY ['SEMIANNUALLY']::data_frequency_type[]` would make me think we'd always set an array with one item in it

Reply:
I think we'd discussed holding off on the semiannual / biannual changes for now - if this proves to be a big lift, we can always just leave the enum on the backend and have FE translate it - this doesn't need to be changed on the backend if it involves a big lift

Reply:
You're correct, this will only set a fixed array and must be updated. Apologies for the oversight. I thought the discussion was that we would not replace the enum type but would update the enum values.

I have code for updating the enum array and am testing it.